compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/AbstractParserErrorHandler.java

```diff
@@ -225,6 +225,15 @@ protected ParserRuleContext getGrandParentContext() {
         return grandParent;
     }
 
+    protected ParserRuleContext getGreatGrandParentContext() {
```

Comment:
Lets see if we can remove this

Reply:
yah. Haven't pushed the commit with code review suggestions yet.